src/_assets/css/_toc.scss

```diff
@@ -0,0 +1,65 @@
+#site-toc--side {
+  $padding: map-get($spacers, 2);
+
+  display: none;
+  right: 0;
+
+  .nav { display: block; }
+  .nav-link {
+    padding-bottom: $padding;
+
+    color: theme-color('primary');
+    &.active {
+      color: theme-color('secondary'); // $gray-dark;
+      font-weight: 500;
+    }
+  }
+
+  > ul > li {
+    padding-bottom: $padding; // Extra space for top-level entries
+    li { padding-left: $padding; } // Indent subentries
+  }
+}
+
+.site-toc {
+  $line-height: normal;
+
+  @include media-breakpoint-up(xl) {
+    padding-bottom: $site-content-top-padding;
+    padding-top: $site-content-top-padding;
+
+    @at-root {
+      #site-toc--inline { display: none; }
+      #site-toc--side { display: block; }
+    }
+  }
+
+  .nav-item {
+    font-size: $font-size-sm;
+
+    .nav-link {
+      color: $site-color-body-light;
+      line-height: $line-height;
+      padding: 0;
+    }
+  }
+
+  &__title {
+    font-family: $site-font-family-alt;
+    font-size: $font-size-lg;
+    margin-bottom: bs-spacer(2);
+  }
+
+  &--button__page-top {
+    @include float-right;
+    font-size: $font-size-sm;
+    line-height: $line-height;
+    padding-top: 0;
+    padding-right: 0;
+
+    &::after {
+      content: 'vertical_align_top'; // 'keyboard_arrow_up';
```

Comment:
The end-of-the-line comments bother me a little. Are they documentation or deletions?

If they're documentation, I'd put them on a separate line and remove the `;`.

If they're deletions, I'd remove them completely.

No need to do this in the current PR, though.

Reply:
I'd defer this, as the file will be moved to shared.